docs/source/release/v6.1.0/mantidworkbench.rst

```diff
@@ -8,53 +8,74 @@ Mantid Workbench Changes
+
+.. figure:: ../../images/MemoryWidget.png
+   :class: screenshot
+   :width: 300px
+   :align: center
+
+Interfaces
+##########
 - Add a checkbox to freeze the rotation in the instrument viewer in Full 3D mode.
-- Calling python's `input` now raises an input dialog in the script editor and the iPython shell.
-- A new empty facility with empty instrument is the default facility now, and
-  user has to select their choice of facility (including ISIS) and instrument for the first time.
-- Added an algorithm ProfileChiSquared1D to profile chi squared after a fit. This can be used
-  to find better estimates of parameter errors.
-- Instrument view: when in tube selection mode, the sum of pixel counts is now output to the selection pane.
-- Added legend property commands to the plot script generator.
-- Added memory widget to display total memory usage. This means that your widget layout will be reset when starting workbench v6.1.0 for the first time. Previously saved layouts accessible from ``View > User Layouts`` may need to be saved again to include the memory bar widget.
+- In instrument view when in tube selection mode, the sum of pixel counts is now output to the selection pane.
+- Automatic data export in DrILL now creates a header for ASCII files, through the new property in :ref:`SaveAscii <algm-SaveAscii>`. See the :ref:`SaveAscii <algm-SaveAscii>` for more information.
 
 
+Plotting
+########
+- New plot interactions: Double click a legend to hide it, double click a curve to open it in the plot config dialog.
+- It is now possible to overplot bin data from the matrix workspace view.
+- When a workspace is renamed it now updates the relevant plot labels with the new workspace name.
+- Added legend property commands to the plot script generator.
+
 Bugfixes
 --------
 
+General
+#######
 - Display Debug and Information messages generated during workbench start up
-- The background shell of spherical and elliptical peaks is now correctly plotted when viewing slices that do not cut through the peak center
-- For the elliptical shell of integrated peaks, the background is correct when plotting with varying background thicknesses
-- Fixed a bug which occurred when switching to a log scale in sliceviewer with negative data.
+- If the facility in `Mantid.user.properties` is empty, it is consistently reflected as empty in the GUI
+- First time dialog box will not appear recurrently, if user selected their choice of facility and instrument at least once and checked "Do not show again until next version".
+- Fixed a bug where Workbench would hang on startup when running on Big Sur.
+- Fixed a bug where TableWorkspace column names would not update correctly if the table was open.
+- Fixed a bug applying constraints with the conjugate gradient minimizer.
+
+Interfaces
+##########
 - Fixed a bug that use wrong help links in certain interfaces
```

Comment:
I'm not sure what this means?

Reply:
Have updated to now say 
`- Correct help links in certain interfaces.`